lib/katsu/docker-compose.yml

```diff
@@ -1,12 +1,12 @@
-version: '3.7'
+version: '4.0'
```

Comment:
Are you sure we should bump this here? Every other docker-compose uses 3.7. If we bump this, we should bump it everywhere.

Reply:
again, didn't know this, can leave 3.7 if we don't want to bump everywhere